CHANGE_LOG

```diff
@@ -1,7 +1,33 @@
-v0.33.0dev
+v0.33.0rc1
 ----------
 
-In development
+This release upgrades to LLVM 9 and drops support for older LLVM versions.
+
+Pull requests:
+
+* PR #593: Fix CUDA with LLVM9
+* PR #592: Fix meta.yaml
+* PR #591: buildscripts: Unpin wheel
+* PR #590: add python_requires to setup.py
+* PR #586: Update changelog for 0.32.1 on master
+* PR #585: Update changelog for 0.32.1 on release branch
```

Comment:
Do these need to go in ? I removed similar from the Numba change log.

Reply:
right, they don't need to